alpaca_trade_api/stream.py

```diff
@@ -646,7 +647,7 @@ def run(self):
             loop.run_until_complete(self._run_forever())
         except KeyboardInterrupt:
             print('keyboard interrupt, bye')
-            pass
+            sys.exit()
```

Comment:
Hey donno, thanks for the reply. Let's remove this change for now. Otherwise, the other changes look good.

Reply:
@haxdds done!

Reply:
Hi Donno! Thanks for getting back. Please remove all new changes from stream.py, so that stream.py is not included in the files changed within the PR.

Reply:
Reversed all changes even though `pass` will literally do nothing...

Reply:
Thanks! It mostly to just make sure the commit history for each file is clean. Your PR looks good. I will merge it to master. Thank you for contributing!